addOns/accessControl/src/main/java/org/zaproxy/zap/extension/accessControl/AccessControlScannerThread.java

```diff
@@ -153,12 +154,9 @@ private void scanImpl() {
 
         // And set up the state accordingly
         this.setScanMaximumProgress(targetNodes.size() + 1);
-        if (log.isDebugEnabled()) {
-            log.debug(
-                    String.format(
-                            "Starting Access Control scan for %d URLs and %d users",
-                            targetNodes.size(), targetUsers.size()));
-        }
+        log.debug(
+                "Starting Access Control scan for %d URLs and %d users",
```

Comment:
`%d` → `{}`

Reply:
Do we specifically want to avoid formatting? Because the logger supports it? (http://logging.apache.org/log4j/2.x/manual/api.html#Formatting_Parameters)

Reply:
You need to obtain a formatter logger though. I don't think we need to format the values/numbers (we don't do that in other places).

Reply:
Okay.